### PR TITLE
Port Enrico siragusa and belonging apps.

### PR DESCRIPTION
**Pull request by simonsasse**
Unfortunately the dissertation of Enrico Siragusa is not in the publications file we use. So only one publication can be cited right now. We could add a second publication file that contains all other publications (probably not many) or leave it as is.

### CODE REVIEW

**Comment by eseiler** [comment]
> Unfortunately the dissertation of Enrico Siragusa is not in the publications file we use. So only one publication can be cited right now. We could add a second publication file that contains all other publications (probably not many) or leave it as is.

Main site: https://refubium.fu-berlin.de/handle/fub188/1
Dissertation: https://refubium.fu-berlin.de/handle/fub188/11364

Is there an API for that?

**Comment by simonsasse** [comment]
So Anja just added enrico siragusa's phd thesis to the database. For now everything should be fine. We'll see, when the next publication is missing. :)

File: _apps/string-similarity-search-and-join.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+header:
+  app_image: /assets/images/apps/searchjoin.png
+links:
+  download: http://packages.seqan.de/searchjoin/
+  source: https://github.com/seqan/seqan/tree/master/apps/searchjoin
+---
+
+**Abstract:**
+
+We present in this paper scalable algorithms for optimal string similarity search and join. Our methods are variations of those applied in Masai, our recently published tool for mapping high-throughput DNA sequencing data with unpreceded speed and accuracy. The key features of our approach are filtration with approximate seeds and methods for multiple backtracking. Approximate seeds, compared to exact seeds, increase filtration specificity while preserving sensitivity. Multiple backtracking amortizes the cost of searching a large set of seeds. Combined together, these two methods significantly speed up string similarity search and join operations.
```
Comment (eseiler) [inline _apps/string-similarity-search-and-join.md:16]: Can you break for 120?

Comment (eseiler) [reply _apps/string-similarity-search-and-join.md:16]: Also in Yara

File: _people/enrico_siragusa.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Enrico Siragusa
+category: Contributor
```
Comment (eseiler) [inline _people/enrico_siragusa.md:3]:
```suggestion
category: Alumni
```
?

**Review by eseiler — CHANGES_REQUESTED** [review]

File: _apps/string-similarity-search-and-join.md (hunk 1 above)
Comment (simonsasse) [reply _apps/string-similarity-search-and-join.md]: Oh yes, you're right!

File: _people/enrico_siragusa.md (hunk 2 above)
Comment (simonsasse) [reply _people/enrico_siragusa.md]: On the old page he was marked as contributor. I wasn't sure where to put him :)

**Review by eseiler — APPROVED** [review]

**Pull request merged**